bin/tester/runtimes.js

```diff
@@ -89,11 +88,20 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
       );
     }
 
-    console.log(dedent`
-        This may lead to some stories not working right or getting detected by Chromatic
-        We suggest you fix the errors, but we will continue anyway..
-        ${separator}
-      `);
+    if (hasErrors) {
```

Comment:
If they have suppressed errors we probably need to say something here.

I wonder if we should just drop the silent behaviour? Is it useful to not show errors if we are going to stop?

Reply:
OK, I took a look and silent is only used via the `DISABLE_LOGGING` env var, which we use in tests, so we don't need to worry to much about it.

Reply:
cool